tests/integration/__init__.py

```diff
@@ -81,7 +81,7 @@
     # Avoid ${TMPDIR} and gettempdir() on MacOS as they yield a base path too long
     # for unix sockets: ``error: AF_UNIX path too long``
     # Gentoo Portage prefers ebuild tests are rooted in ${TMPDIR}
-    os.environ.get('TMPDIR', tempfile.gettempdir()) if salt.utils.is_darwin() else '/tmp'
+    os.environ.get('TMPDIR', tempfile.gettempdir()) if not salt.utils.is_darwin() else '/tmp'
```

Comment:
We can do that if you want, but it is my fault the line reads as it currently does.  Prior to my [previous change](https://github.com/saltstack/salt/pull/32447/files), this was the behavior and I had inadvertently inverted the logic.